modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ButtonTag.java

```diff
@@ -70,6 +72,10 @@ public String getIcon() {
 		return _icon;
 	}
 
+	public boolean getIconRight() {
+		return _iconRight;
+	}
+
```

Comment:
A question about the naming? Shouldn't this be "sideIcon"? It's going to be on the right for LR languages, but for RL languages it's going to be in the opposite side, right? @ambrinchaudhary

Reply:
That's a good point @adolfopa. I suppose nobody thought on that, as for now the icon was always on the left. 
But this is set by code, when the taglib is used. If it should depend on the language, then I guess it should be read by a settings value of something similar, shouldn't be? Let me ask Maria for confirmation.

Reply:
We have seen that most of the content changes direction when you set the page in a RL Language, so we should do the same with this dropdown. 
![Screenshot 2023-09-11 at 11 18 43](https://github.com/liferay-lima/liferay-portal/assets/8373764/9138d03e-5f77-47d9-9a84-cee9642bb8d3)
![Screenshot 2023-09-11 at 11 16 30](https://github.com/liferay-lima/liferay-portal/assets/8373764/47cf2fa0-cda8-4606-9e8d-d30f1eb7568a)

Said that, @adolfopa do you have any idea how this is currently done? from where you can read if it is LR or RL language?

Reply:
No idea, sorry. You may need to ask someone with knowledge about accessibility in the portal.

Reply:
No worries, we found that this is magically done with the dir="rtl" attribute set in the <html> tag, so it will be put automatically on the left side when RTL is set. 
Said that... I don't think is a bad idea to name it `iconRight` (or something similar), thinking that the LTR is the default mode. I prefer that than using a `sideIcon` with strings for left or rigth. How does it sound to you?

Reply:
Not sure about this. Using `iconRight` may be confusing, as the icon isn't necessarily on the right depending on the user (or developer's) locale. I don't have a strong opinion (i.e. I don't really care), but I think it is cleaner to have a name that works all the time, and not only in the default case.

Reply:
Maybe we should ask to someone in frontend-infra/peds; they must have encountered things like this.

Reply:
In any case, as long as you promise to ask (and change things if necessary later), I'm fine with moving this forward.

Reply:
this pull must be reviewed by FI, so yes, they will be asked ;-)

Reply:
see https://github.com/liferay-frontend/liferay-portal/pull/3649 :smiley: